src/_community/openings.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Openings
+description: Current Openings for positions on the Quilt Staff Team
+
+redirect_from:
+  - /community/openings.html
+---
+The Quilt Project is very community-oriented, and that means that it needs your help to succeed. This page is a list of all the places we currently need help with, both in development and in our community.
```

Comment:
"All the places"?

Reply:
The language may need work

Reply:
I think the original wording was fine, you could've just changed it to "this page" and it'd work imo.